ffi/rtc.lua

```diff
@@ -302,6 +304,13 @@ function RTC:validateWakeupAlarmByProximity(task_alarm, proximity)
             "\ncurrent time is     " .. now .. os.date(" (%F %T %z)", now))
     end
 
+    -- On dodgy RTCs, some aging batteries are (supposedly) causing reads to report a bogus value...
+    -- c.f., https://github.com/koreader/koreader/issues/7994 & https://github.com/koreader/koreader/issues/10996
+    if self.dodgy_rtc and alarm_sys <= 1 then
+        print("Read back a bogus alarm value from a dodgy RTC, assuming our previously set alarm fired as expected anyway")
```

Comment:
The phrasing seems a little unclear to me (if I saw it in the logs). Maybe the other way around? "A dodgy RTC…"

Reply:
Done ;).

FWIW, it's never shown in a vacuum, it's part of the whole `validate...` block, e.g.,

```
validateWakeupAlarmByProximity:	
task              @ 1697239968 (2023-10-14 01:32:48 +0200)	
last set alarm    @ 1697239968 (2023-10-14 01:32:48 +0200)	
current rtc alarm @ 1 (1970-01-01 01:00:01 +0100)	
current rtc time is 1697239983 (2023-10-14 01:33:03 +0200)	
current time is     1697239983 (2023-10-14 01:33:03 +0200)
Read back a bogus alarm value from a dodgy RTC, assuming our previously set alarm fired as expected anyway
```

(That was yet another older version of said message, though ;p).